packages/expo-ads-facebook/src/AdSettings.ts

```diff
@@ -47,6 +47,10 @@ export default {
    * Sets URL prefix
    */
   setUrlPrefix(urlPrefix: string): void {
-    CTKAdSettingsManager.setUrlPrefix(urlPrefix);
+    let nullableUrlPrefix: string | null = urlPrefix;
+    if (nullableUrlPrefix && nullableUrlPrefix.length === 0) {
```

Comment:
Remove this - this check will never pass unless `nullableUrlPrefix` is an empty array (not an empty string).